### DISCUSSION TITLE
Новые блоки.

### DISCUSSION THREAD

**Discussion by SIANIE19HW** [Sokol-Games-GmbH/PixelCombats.PublicApi#212] [Ideas]
Здравствуйте я хочу вам преподнести идею многих игроков о новых блоках для игры.
1. Цветы. В Игре есть растения, трава, и дерево но для постройки например леса или как декоративной вещи подойдут именно цветы
2. Лампочки. Подойдёт для декораций.
P.s... Извините что так мало.

**Comment by kkohno** [comment]
подумаем над Вашим предложением. Возможно добавим блоков
